File: src/ProjectR/content/scripts/spells/HealingTouch.lua
function GetTargetType()
    return Single
end

function GetName()
    return "Healing Touch"
end

function GetDescription()
    return "Slighty heals your target and restores some of your MP."
end

function IsSupportSpell()
    return true
end

function GetMPCost()
    return 0
end

function GetDelay()
  return .5
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
    d:Heal(aAD + aMD)
    a:UseMP(-(aMP * .2))
end

File: src/ProjectR.Scripting/content/scripts/spells/HealingTouch.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting
{
    public class HealingTouch : SpellScriptBase
    {
        public override string Name { get { return "Healing Touch"; } }
        public override string Description { get { return "Slighty heals your target and restores some of your MP."; } }

        public override TargetType TargetType { get { return TargetType.Single; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Composite; } }
        public override bool IsSupportSpell { get { return true; } }

        public override double MPCost { get { return 0; } }
        public override double Delay { get { return .5; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            Heal(cAD + cMD);
            caster.UseMP(-(cMP * .2));
        }
    }
}